MobiFlight/MidiBoard/MidiBoardManager.cs

```diff
@@ -29,40 +34,33 @@ public MidiBoardManager ()
         private List<string> CollectErrorsInInputDefinition(MidiBoardDefinition def)
         {
             List<string> errorMessages = new List<string>();
-            if (def.Inputs == null)
-                errorMessages.Add($"No input definitions.");
-            else
+
+            foreach (var item in def.Inputs)
             {
-                foreach (var item in def.Inputs)
-                {
-                    if (item.LabelIds.Length != item.MessageIds.Length)
-                        errorMessages.Add($"{item.Label}: Number of LabelIds unequal to number of MessageIds");
-                    if (!item.Label.Contains("%"))
-                        errorMessages.Add($"{item.Label}: Label does not contain % placeholder.");
```

Comment:
This is now enforced via the schema.

Reply:
Will these messages also become visible somewhere?

Reply:
They get logged in CollectAndLogDefinitionErrors(). I didn't change that part of the code.